release notes/v0.4.0.md

```diff
@@ -0,0 +1,117 @@
+
+- Add `Add locator.isEditable` which will return `true` or `false` if an uniquely identifiable element on the page is editable (e.g. `<input>`) ([#372](https://github.com/grafana/xk6-browser/pull/372)).
+
+- Add `locator.isEnabled` which will return `true` or `false` if an uniquely identifiable element on the page is enabled ([#373](https://github.com/grafana/xk6-browser/pull/373)).
+
+- Add `locator.isDisabled` which will return `true` or `false` if an uniquely identifiable element on the page is disabled ([#375](https://github.com/grafana/xk6-browser/pull/375)).
+
+- Add `locator.isVisible` which will return `true` or `false` if an uniquely identifiable element is visible ([#376](https://github.com/grafana/xk6-browser/pull/376)).
+
+- Add `locator.isHidden` which will return `true` or `false` if an uniquely identifiable element is hidden ([#377](https://github.com/grafana/xk6-browser/pull/377)).
+
+- Add `locator.fill` will fill an input with the supplied text ([#380](https://github.com/grafana/xk6-browser/pull/380), [#383](https://github.com/grafana/xk6-browser/pull/383)).
+
+- Add `locator.focus` will focus on an uniquely identifiable element ([#388](https://github.com/grafana/xk6-browser/pull/388)).
+
+- Add `locator.getAttribute` will return the named attribute of a selected locator ([#390](https://github.com/grafana/xk6-browser/pull/390)).
+
+- Add `locator.innerHTML` which will return the html of the given selector ([#392](https://github.com/grafana/xk6-browser/pull/392)).
+
+- Add `locator.innerText` will return the `innerText` from the locator ([#393](https://github.com/grafana/xk6-browser/pull/393)).
+
+- Add `locator.textContent` will return the text from the locator ([#394](https://github.com/grafana/xk6-browser/pull/394)).
+
+- Add `locator.inputValue` will return the text inputted in an input field ([#395](https://github.com/grafana/xk6-browser/pull/395)).
+
+- Add `locator.selectOption` will select the given option in a select element ([](https://github.com/grafana/xk6-browser/pull/397)).
+
+- Add `locator.press` which will "push" the specified keyboard key ([#398](https://github.com/grafana/xk6-browser/pull/398)).
+
+- Add `locator.type` which will "type" the text into the input element ([#399](https://github.com/grafana/xk6-browser/pull/399)).
+
+- Add `locator.hover` which will bring into view the selected element ([#400](https://github.com/grafana/xk6-browser/pull/400)).
+
+- Add `locator.tap` which will bring the into view the selected element and tap on it ([#401](https://github.com/grafana/xk6-browser/pull/401)).
+
+- Add `locator.dispatchEvent` which will dispatch the specified event to the selected element ([#405](https://github.com/grafana/xk6-browser/pull/405)).
+
+- Add `locator.waitFor` which will wait for the element selected to meet the specified state ([#406](https://github.com/grafana/xk6-browser/pull/406)).
+
+
+## Bugs fixed
```

Comment:
#401's https://github.com/grafana/xk6-browser/commit/5e1eb9eb930e3b0ebcffe097eb115ad2604de217 is missing.

#382 is missing.

Reply:
- #401 is in the New Features section
- #382 is covered by the PR#383 and is in the New Features section

Reply:
- #401 is in New Features
- #382 is covered by #383 in New Features

Reply:
- https://github.com/grafana/xk6-browser/pull/401 is in New Features
- https://github.com/grafana/xk6-browser/issues/382 is covered by https://github.com/grafana/xk6-browser/pull/383 in New Features

Reply:
- https://github.com/grafana/xk6-browser/pull/401 is in New Features
- https://github.com/grafana/xk6-browser/issues/382 is covered by https://github.com/grafana/xk6-browser/pull/383 in New Features

Reply:
Ah, OK. But, there was also a bug fix for `ElementHandle.Tap`. I think we should put it in the bug fixes section—it wasn't working before. People should know about it.

#383: Yep, I know, sorry, I misphrased it 🤦 I mean to put it in the bug section as well.